ignition-common-3/PKGBUILD

```diff
@@ -12,10 +12,17 @@ depends=('ignition-math>=6' 'tinyxml2' 'freeimage' 'libutil-linux' 'gts' 'ffmpeg
 makedepends=('ignition-cmake>=2' 'util-linux')
 optdepends=()
 provides=("ignition-common=$pkgver")
-source=("https://github.com/ignitionrobotics/ign-common/archive/ignition-common3_${pkgver}.tar.gz")
-sha256sums=('90b5d0d37c17ea92becbf3e339d99ce777336ad076e3c9550340e3ad33b4ff52')
+source=("https://github.com/ignitionrobotics/ign-common/archive/ignition-common3_${pkgver}.tar.gz"
+        "missing-include-cstring.patch")
+sha256sums=('293ff4d9e4e5814535ddc40b45794c05153e43b274cee25744a3eea493da5641'
+            '9b9047fd4c8f2de5aabfeab36cc64062d6d30564d7449a5164715f6ac1ff64d0')
 
-_dir="ign-common-ignition-common3_${pkgver}"
+_dir="gz-common-ignition-common3_${pkgver}"
+
+prepare() {
+  cd "$srcdir/$_dir"
+  patch -p1 < ../missing-include-cstring.patch
```

Comment:
Can you make this an upstream patch like so:
https://github.com/acxz/gazebo-arch/blob/74501764bd86da0f58475ebb2750af5525fd2e26/ignition-common/PKGBUILD#L15